google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -118,6 +120,11 @@ public void onOutputLine(String output) {
         deployLogListener,
         deployOutputListener,
         deployExitListener);
+
+    // show a warning notification if the cloud sdk version is not supported
+    new CloudSdkVersionCheck().performCloudSdkVersionCheck(
```

Comment:
I wonder if there is a way to get a handle on the singleton, instead of instantiating a new one, since we are already using the intellij container to create this class as a StartupActivity.